src/cli/test/create.js

```diff
@@ -104,6 +110,13 @@ const builder = {
   connection: {
     describe: 'Choose the emulated network connection speed.'
   },
+  webhookUrl: {
+    describe: 'Test result JSON will be POSTed to this URL.'
```

Comment:
I think I know what this means,  but still, I wouldn't abbreviate the explanation. How about `Test result JSON will be sent to this URL via the HTTP POST method.`?